### PR TITLE
adds website header to all website related pages

### PR DESCRIPTION
**Pull request by levibrown**
@lance-hardy this is looking pretty good, but needs a few adjustments

### CODE REVIEW

**Comment by lance-hardy** [comment]
@levibrown  If you merge I'll pull down and check it.

File: app/templates/components/website-row.hbs [hunk 1]
```diff
@@ -10,11 +10,13 @@
   <div class="chip status">
     {{fa-icon statusIcon}} <span>{{website.status}}</span>
   </div>
-  {{#if website.canEdit}}
-    <!-- <a href="" class="btn btn--a btn-edit" style="color: red;">{{fa-icon "edit"}} Fix</a> -->
-    {{#link-to "website.index" website class="btn" }}{{fa-icon "edit"}} Edit{{/link-to}}
-  {{else}}
-    <a href="#" class="btn secondary-content">{{fa-icon "edit"}} Edit</a>
+  {{#if (not header)}}
```
Comment (variousred) [inline app/templates/components/website-row.hbs]: so many conditionals 😖

**Comment by levibrown** [comment]
@lance-hardy can you just use this branch? it has current ember2-upgrade merged in

**Comment by lance-hardy** [comment]
@levibrown yeah, I'll pull it down

**Comment by lance-hardy** [comment]
@levibrown location header work got included here but that's cool. website row markup and styles dialed.

**Pull request merged**